stripe/src/test/java/com/stripe/android/paymentsheet/PaymentOptionsViewModelTest.kt

```diff
@@ -164,5 +201,16 @@ class PaymentOptionsViewModelTest {
         private val SELECTION_SAVED_PAYMENT_METHOD = PaymentSelection.Saved(
             PaymentMethodFixtures.CARD_PAYMENT_METHOD
         )
+        private val paymentMethodCreateParams: PaymentMethodCreateParams = mock()
```

Comment:
You can use `PaymentMethodCreateParamsFixtures.DEFAULT_CARD`

Also should be rename to `PAYMENT_METHOD_PARAMS`

Reply:
You are thinking PaymentMethodCreateParamsFixtures.PAYMENT_METHOD_PARAMS?  The DEFAULT_CARD does kind of help indicate that it has default card details in them.

Reply:
Oh I mean rename the variable from `paymentMethodCreateParams` to `PAYMENT_METHOD_PARAMS` or something like that

Reply:
got it